universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -118,6 +123,15 @@ public boolean hasValueAtPath(Path path) {
     }
   }
 
+  public void putDate(Path path, String dateString) {
```

Comment:
(blocking) this should have a comment on about the required "yyyy-MM-dd" format.